redis/redis/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
+    networks: 
+      tut-redis: 
+        ipv4_address: 172.19.0.3
+    volumes:
+    - /var/log/tut-redis/gateway:/kaazing-gateway/log
+  redis:
+    build: broker
+    ports:
+      - "6379:6379"
```

Comment:
To be honest, this depends on the design of the system. Assuming that gateway is the only client of the redis, your comment is correct. I assumed that different clients might be required to connect to redis, other than the gateway. I will remove it, since I think this is not the case.